### ISSUE TITLE
[UAT site] Synchronisation between maps and list on the left is not working

### ISSUE THREAD

**Issue by hohangleung** [MSFREACH/MSF-REACH-COLLABORATION#123]
That will conflict with the Search and Type Filter

**Comment by LucieGueuning** [comment]
No. Map is an additional filter to be considered. It will help narrowing down the results of the list on the left. 

So compilation of map as filter, status, type and search filter. It is like you are searching for 'dengue fever' in Indonesia (thanks to map).
